docs/common/dockerhub-ratelimit.md

```diff
@@ -1,9 +0,0 @@
-# Verify your quota limit in dockerhub for your user
```

Comment:
This might be useful in some contexts, e.g. while debugging stuffs

Reply:
Did we use it in the last 2 years? :-D